### ISSUE TITLE
Clarify gains parameter in README

### ISSUE THREAD

**Issue by alexagv** [openDAQ/python-opendaq#1]
Hi, it's not very clear in the example in the README nor on [readthedocs](https://opendaq-python.readthedocs.io/en/latest/usage.html?highlight=conf_adc#adc-reading-command-response-mode) that gain needs to be set according to the openDAQ model (Gains.N.x1 for model N, Gains.S.x1 for model S etc)
The example in the README should probably be updated to `Gains.N.x1` as N is the latest model?

**Comment by joserb** [comment]
Yes, thanks for letting us know. Actually, much of the documentation needs to be updated. 
In any case, do not hesitate to contact us directly if you have any questions.
